mautic/mautic
# Hide the input field for the batch size

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [yes]
| Deprecations?                          | [x]
| BC breaks? (use the c.x branch)        | [x]
| Automated tests included?              | [x] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


The input field for the batch size on the email sending screen was hidden. See https://github.com/mautic/mautic/pull/10967 for previous discussions leading up to this decision.

![image](https://user-images.githubusercontent.com/96085911/168838759-06b0c626-1973-4d5b-b3a3-91834e839705.png)

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create new segment email
3. Try to send it
4. The input field is hidden




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11167"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
